vendor/bat-native-ledger/src/bat/ledger/internal/wallet_provider/transfer.cc

```diff
@@ -0,0 +1,115 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "bat/ledger/internal/wallet_provider/transfer.h"
+
+#include <utility>
+
+#include "base/guid.h"
+#include "base/strings/string_number_conversions.h"
+#include "bat/ledger/internal/ledger_impl.h"
+
+namespace ledger::wallet_provider {
+
+Transfer::Transfer(LedgerImpl* ledger) : ledger_(ledger) {
+  DCHECK(ledger_);
+}
+
+Transfer::~Transfer() = default;
+
+void Transfer::Run(const std::string& contribution_id,
+                   const std::string& destination,
+                   double amount,
+                   ledger::ResultCallback callback) const {
+  MaybeCreateTransaction(
+      contribution_id, destination,
+      // We can one-time/monthly tip in multiples of 0.25.
```

Comment:
Instead of documenting the current state of things (which will change), can we instead document:

* That we're rounding to `0.0001` (as a business rule). You already have this.
* The minimum tip amount that this will support, given a 5% fee (which I guess is `0.002`?)

Reply:
Sure, changed to:
```
// Rounding to nearest ten-thousandth (0.0001),
// which supports a minimum tip amount of 0.002 (given a 5% fee).
```